Plugin/Magento/Sales/Api/Data/OrderExtension.php

```diff
@@ -85,13 +83,23 @@ public function afterGetDeliveryOptions()
     }
 
     /**
-     * @return mixed
+     * @param int $entityId
+     *
+     * @return array
+     */
+    private function getIdByEntityId(int $entityId): array
```

Comment:
Ik snap dat je wilt dat het hetzelfde returned als die eronder, maar dit is wel een rare functie 

Waarom heten deze überhaupt "getId..."? Hoe is zo'n `[$searchColumn, $searchValue]` array een id?

Reply:
Done